packages/p2p/src/pages/my-profile/my-profile.tsx

```diff
@@ -45,11 +45,7 @@ const MyProfile = () => {
         }
     }, [is_poi_poa_verified]);
 
-    if (
-        isEmptyObject(general_store.advertiser_info) &&
-        !general_store.poi_status &&
-        !general_store.should_show_dp2p_blocked
-    ) {
+    if (general_store.is_p2p_user === null) {
```

Comment:
Since we no longer call the endpoint, the previous checks will forever load the page, needed to check if get_account_status returns p2p_status instead. Resolves loading issue